lib/facter/simplib__numa.rb

```diff
@@ -0,0 +1,78 @@
+# _Description_
+#
+# Return true if system has multiple NUMA nodes
+#
+Facter.add("simplib__numa_enabled") do
+  confine :kernel => 'Linux'
+  confine { File.exist? '/sys/devices/system/node' }
+
+  setcode do
+    true
+  end
+
+end
+
+# _Description_
+#
+# Return supported NUMA possible nodes
+#
+Facter.add("simplib__numa_possible_nodes") do
+  confine :kernel => 'Linux'
+  confine { File.exist? '/sys/devices/system/node/possible' }
+
+  # read the file
+  File.open('/sys/devices/system/node/possible', 'r') do | text |
```

Comment:
This can be shortened to `File.read('/sys/devices/system/node/possible').strip` (you want to chop off leading and ending spaces